src/ape_ethereum/transactions.py

```diff
@@ -186,6 +189,41 @@ class SharedBlobTransaction(DynamicFeeTransaction):
     """
 
 
+class Authorization(BaseModel):
+    """
+    `EIP-7702 <https://eips.ethereum.org/EIPS/eip-7702>`__ authorization list item.
+    """
+
+    chain_id: HexInt = Field(alias="chainId")
+    address: AddressType
+    nonce: HexInt
+    v: HexInt = Field(alias="yParity")
```

Comment:
Option: we could have a model validator make a message signature and model serializer serialize back to v,r,s in the JSON

Reply:
I was considering this, but to be honest I would rather have the constituent values, and then compute the signature on the fly when needed